test-graalvm/common/src/main/java/testgraalvm/domain/Owner.java

```diff
@@ -29,6 +29,20 @@ public class Owner {
     private String name;
     private int age;
 
+    public Owner() {
+    }
+
+    public Owner(String name, int age) {
+        this.name = name;
+        this.age = age;
+    }
+
+    public Owner(Long id, String name, int age) {
```

Comment:
should this contractor be annotated with `@Creator`?

Reply:
Not needed